facts/Vagrantfile

```diff
@@ -78,30 +78,30 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision 'shell', path: 'get_facts.sh'
     host.vm.provision 'shell', inline: '/sbin/shutdown -h now'
   end
-  # images are at https://cloud.centos.org/centos/9-stream/x86_64/images/
-  # can be imported with `vagrant box add --name centos/stream9 $url` you can also
-  # use the host.vm.box_url below
-  # facts land as centos-9-x86_64.facts
-  #
   config.vm.define 'centos-stream9-x86_64', autostart: false do |host|
     host.vm.box = 'centos/stream9'
-    host.vm.box_url = 'https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-Vagrant-9-20240513.0.x86_64.vagrant-virtualbox.box'
-    # centos packages their box with synced folders set to rsync (presumably
-    # since vboxsf is not built in their kernel)
-    # If the version of the kernel-headers/kernel-devel that are available
-    # for download are newer than what is on the box then guest additions will
-    # not build. If the stars align, then you can set `type: 'virtualbox'`
-    # for synced_folder and maybe it will work, otherwise, it is a manual
-    # effort to rsync the fact files back
-    # host.vm.synced_folder '.', '/vagrant', type: 'virtualbox'
+    host.vm.box_url = 'https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-Vagrant-9-latest.x86_64.vagrant-virtualbox.box'
+    if ENV['mount_synced_folder']
```

Comment:
do we need to set this environment variable?

Reply:
read all, ask then ;) see readme change ;)

Reply:
and to answer properly: yes, we need to provision runs to get the centos facts.